tinygrad/ops_cpu.py

```diff
@@ -109,23 +103,20 @@ def forward(ctx, x, shape):
 
   @staticmethod
   def backward(ctx, grad_output):
-    in_shape, = ctx.saved_tensors
-    return grad_output.reshape(in_shape)
+    return grad_output.reshape(*ctx.saved_tensors)
 register('reshape', Reshape)
 
 
 # ************* activation ops *************
 
 class ReLU(Function):
   @staticmethod
-  def forward(ctx, input):
-    ctx.save_for_backward(input)
-    return np.maximum(input, 0)
+  def forward(ctx, _input):
+    return np.maximum(*ctx.save_for_backward(_input), 0)
```

Comment:
how can this work? does save_for_backward return anything? may be now

Reply:
Yeah, I updated it to have it return its params - that's how all of my updates work